src/Valleysoft.DockerRegistryClient/Models/OciAnnotations.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Valleysoft.DockerRegistryClient.Models;
+
+/// <summary>
+/// Pre-defined annotation keys from the OCI spec.
+/// </summary>
+public static class OciAnnotations
```

Comment:
Missing:
- `org.opencontainers.artifact.created`
- `org.opencontainers.artifact.description`
- `org.opencontainers.image.base.digest`
- `org.opencontainers.image.base.name`
- `org.opencontainers.referrers.filtersApplied`

Reply:
I've only added annotations that are defined in the 1.0 spec. The ones you mentioned are in 1.1 which has not yet been released. I'm going to hold off on those until 1.1 is officially released since they may be subject to change.